cogs/subscribers.py

```diff
@@ -34,7 +34,6 @@
 # Subscriber decorator
 from .utils.subscribers import canary_subscriber
 
-CFIA_FEED_URL = "http://inspection.gc.ca/eng/1388422350443/1388422374046.xml"
```

Comment:
why is this removed?

Reply:
i think it was a typo or something, i dont think it was intended